unity/Assets/Scripts/CollisionListener.cs

```diff
@@ -31,8 +31,20 @@ public class StaticCollision
             }
         }
     }
+
     private HashSet<Collider> activeColliders = new HashSet<Collider>();
+    public static bool useMassThreshold = false;
+    public static float massThreshold = 10f;
+
+    public void Set_massThreshold (float f)
```

Comment:
Can you remove the underscore and rename to SetMassThreshold, same for Set

Reply:
removing these since it would be better to set the values directly as mentioned above.